lib/cog/adapter.ex

```diff
@@ -45,7 +45,8 @@ defmodule Cog.Adapter do
       end
 
       defp payload(sender, room, text) do
-        %{sender: sender,
+        %{id: UUID.uuid4(:hex),
```

Comment:
This is the format we use elsewhere for such IDs. Need to do another PR to consolidate those calls a bit.